steps/sonarqube-msbuild-scanner/1.0.0/step.yml

```diff
@@ -0,0 +1,113 @@
+title: SonarQube MSBuild Scanner
+summary: |
+  Step for running SonarQube MSBuild Scanner CLI.
+description: |
+  Step for running SonarQube MSBuild Scanner CLI.
+website: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner
```

Comment:
website, source_code_url, support_url points to a nonexistent location. There is an extra dash in `sonar-qube`.